pontoon/base/migrations/0018_populate_entity_context.py

```diff
@@ -3,22 +3,20 @@
 from django.db import migrations
 from django.db.models import F, Func, TextField, Value
 
-from pontoon.sync import KEY_SEPARATOR
-
 
 def add_entity_context(apps, schema_editor):
     Entity = apps.get_model("base", "Entity")
 
     split_key_po = Func(
         F("key"),
-        Value(KEY_SEPARATOR),
+        Value("\x04"),
```

Comment:
Raising this again, because I'm curious about the change.

Reply:
Sorry, missed this earlier. I found `x04` to be actually easier to search for, as that's how it's also used within translate-toolkit code. This will also go away with the message data model refactor.